File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int, int> mp;
        for(int x : nums) mp[x]++;
        for(int x : nums) {
            if (mp[x] > 1) return true;
        }
        return false;

    }
};

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        for i, n in enumerate(nums):
            need = target - n
            if need in hashmap:
                return [hashmap[need] + 1, i + 1]
            hashmap[n] = i
        